test/integration/content-preview/Annotations.e2e.test.js

```diff
@@ -0,0 +1,94 @@
+// Specify Annotations version in command line, otherwise defaults to the latest version
+// e.g. CYPRESS_ANNOTATIONS_VERSION=4.0.0-beta.24 yarn test:e2e:open
+const ANNOTATIONS_VERSION = Cypress.env('ANNOTATIONS_VERSION') ? `@${Cypress.env('ANNOTATIONS_VERSION')}` : '';
+
+import(`https://unpkg.com/box-annotations${ANNOTATIONS_VERSION}/dist/annotations.js`);
+
+describe('ContentPreview with Annotations', () => {
+    const helpers = {
+        load({ features, fileId, props } = {}) {
+            cy.visit('/Elements/ContentPreview', {
+                onBeforeLoad: contentWindow => {
+                    contentWindow.FEATURES = {
+                        activityFeed: {
+                            annotations: {
+                                enabled: true,
+                            },
+                        },
+                        ...features,
+                    };
+                    contentWindow.FILE_ID = fileId;
+                    contentWindow.PROPS = {
+                        showAnnotations: true,
+                        ...props,
+                    };
+
+                    const { document } = contentWindow;
+                    const { head } = document;
+
+                    const link = document.createElement('link');
+                    link.rel = 'stylesheet';
+                    link.type = 'text/css';
+                    link.href = `https://unpkg.com/box-annotations${ANNOTATIONS_VERSION}/dist/annotations.css`;
+                    head.appendChild(link);
+                },
+            });
+        },
+        clickAnnotationLinkByComment(comment) {
+            cy.contains(comment)
```

Comment:
Maybe we could `return` the subject here as well so we can possibly chain any selected state expectations in the future?